bin/fuel-core/src/cli/run.rs

```diff
@@ -194,6 +194,10 @@ pub struct Command {
     /// Time to wait after submitting a query before debug info will be logged about query.
     #[clap(long = "query_log_threshold_time", default_value = "2s", env)]
     pub query_log_threshold_time: humantime::Duration,
+
+    /// Timeout before drop the request.
+    #[clap(long = "api-request-timeout", default_value = "30m", env)]
```

Comment:
30s is too short, as it may take longer than that for transactions to make it into a block and users will have a subscription open during that whole duration.